Listening for transport dt_socket at address: 9001
 INFO 15:44:06,735 JNA not found. Native methods will be disabled.
DEBUG 15:44:06,751 Loading settings from /home/jinsu/research/cass-prj/trunk/stable_cass/conf/conf1/storage-conf.xml
DEBUG 15:44:06,984 Syncing log with a period of 10000
 INFO 15:44:06,989 DiskAccessMode 'auto' determined to be standard, indexAccessMode is standard
DEBUG 15:44:07,069 setting autoBootstrap to false
DEBUG 15:44:07,110 setting hintedHandoffEnabled to true
DEBUG 15:44:07,282 adding Super1 as 0
DEBUG 15:44:07,282 adding Standard2 as 1
DEBUG 15:44:07,283 adding Super2 as 2
DEBUG 15:44:07,284 adding Standard1 as 3
DEBUG 15:44:07,284 adding StandardByUUID1 as 4
DEBUG 15:44:07,285 adding LocationInfo as 5
DEBUG 15:44:07,285 adding HintsColumnFamily as 6
DEBUG 15:44:07,415 Starting CFS LocationInfo
DEBUG 15:44:07,433 key cache capacity for LocationInfo is 1
DEBUG 15:44:07,435 Starting CFS HintsColumnFamily
DEBUG 15:44:07,436 key cache capacity for HintsColumnFamily is 1
DEBUG 15:44:07,478 opening keyspace Keyspace1
DEBUG 15:44:07,478 Starting CFS Standard2
DEBUG 15:44:07,479 key cache capacity for Standard2 is 1
DEBUG 15:44:07,480 Starting CFS Super1
DEBUG 15:44:07,480 key cache capacity for Super1 is 200000
DEBUG 15:44:07,481 Starting CFS Standard1
DEBUG 15:44:07,481 key cache capacity for Standard1 is 200000
DEBUG 15:44:07,482 Starting CFS Super2
DEBUG 15:44:07,488 key cache capacity for Super2 is 1
DEBUG 15:44:07,488 row cache capacity for Super2 is 10000
DEBUG 15:44:07,489 Starting CFS StandardByUUID1
DEBUG 15:44:07,490 key cache capacity for StandardByUUID1 is 200000
DEBUG 15:44:07,490 opening keyspace system
 INFO 15:44:07,518 Saved Token not found. Using 115334859289675963285972157264705071312
 INFO 15:44:07,519 Saved ClusterName not found. Using Test Cluster
DEBUG 15:44:07,523 Estimating compactions for Super1
 INFO 15:44:07,530 Creating new commitlog segment /tmp/fi/cassandra/node1/commitlog/CommitLog-1308955447530.log
DEBUG 15:44:07,533 Estimating compactions for Standard2
DEBUG 15:44:07,533 Estimating compactions for Super2
DEBUG 15:44:07,533 Estimating compactions for Standard1
DEBUG 15:44:07,534 Estimating compactions for StandardByUUID1
DEBUG 15:44:07,534 Estimating compactions for LocationInfo
DEBUG 15:44:07,534 Estimating compactions for HintsColumnFamily
DEBUG 15:44:07,535 Checking to see if compaction of Super1 would be useful
DEBUG 15:44:07,535 Checking to see if compaction of Standard2 would be useful
DEBUG 15:44:07,536 Checking to see if compaction of Super2 would be useful
DEBUG 15:44:07,536 Checking to see if compaction of Standard1 would be useful
DEBUG 15:44:07,536 Checking to see if compaction of StandardByUUID1 would be useful
DEBUG 15:44:07,537 Checking to see if compaction of LocationInfo would be useful
DEBUG 15:44:07,537 Checking to see if compaction of HintsColumnFamily would be useful
 INFO 15:44:07,572 LocationInfo has reached its threshold; switching in a fresh Memtable at CommitLogContext(file='/tmp/fi/cassandra/node1/commitlog/CommitLog-1308955447530.log', position=419)
 INFO 15:44:07,577 Enqueuing flush of Memtable-LocationInfo@3743136(169 bytes, 4 operations)
 INFO 15:44:07,579 Writing Memtable-LocationInfo@3743136(169 bytes, 4 operations)
TRACE 15:44:07,695 wrote DecoratedKey(61078635599166706937511052402724559481, L) at 0
TRACE 15:44:07,695 wrote index of DecoratedKey(61078635599166706937511052402724559481, L) at 0
 INFO 15:44:07,704 Completed flushing /tmp/fi/cassandra/node1/data/system/LocationInfo-1-Data.db
DEBUG 15:44:07,712 Checking to see if compaction of LocationInfo would be useful
DEBUG 15:44:07,713 discard completed log segments for CommitLogContext(file='/tmp/fi/cassandra/node1/commitlog/CommitLog-1308955447530.log', position=419), column family 5. CFIDs are {Keyspace1: TableMetadata({Standard2: 1, Super1: 0, Standard1: 3, Super2: 2, StandardByUUID1: 4, }), system: TableMetadata({LocationInfo: 5, HintsColumnFamily: 6, }), }
DEBUG 15:44:07,714 Marking replay position 419 on commit log CommitLogSegment(/tmp/fi/cassandra/node1/commitlog/CommitLog-1308955447530.log)
 INFO 15:44:07,734 Starting up server gossip
- TM uNT(1) : size of sortedToken = 0
- TM uNT(2) : size of sortedToken = 1 ... leavingEndPoints = []
TokenSizeTest:1:-1009577
 INFO 15:44:07,808 Binding thrift service to /127.0.0.11:9160
 INFO 15:44:07,826 Cassandra starting up...
TRACE 15:44:07,874 eof reading from socket; closing
java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:375)
	at org.apache.cassandra.net.IncomingTcpConnection.readInt_aroundBody4(IncomingTcpConnection.java:59)
	at org.apache.cassandra.net.IncomingTcpConnection.readInt_aroundBody5$advice(IncomingTcpConnection.java:236)
	at org.apache.cassandra.net.IncomingTcpConnection.run(IncomingTcpConnection.java:59)
TRACE 15:44:07,875 Received a GossipDigestSynMessage from /127.0.0.13
TRACE 15:44:07,883 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,883 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,888 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,889 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:1
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,895 Received a GossipDigestSynMessage from /127.0.0.12
DEBUG 15:44:07,895 attempting to connect to /127.0.0.13
TRACE 15:44:07,896 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,896 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,897 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,897 Sending a GossipDigestAckMessage to /127.0.0.12
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:2
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,898 Received a GossipDigestSynMessage from /127.0.0.1
TRACE 15:44:07,899 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,899 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,899 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,899 Sending a GossipDigestAckMessage to /127.0.0.1
- Experiment not running, continue ===>GDSynVHandler
DEBUG 15:44:07,900 attempting to connect to /127.0.0.12
Sending message ::
-->ID:3
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,902 Received a GossipDigestSynMessage from /127.0.0.12
TRACE 15:44:07,902 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,903 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,903 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,903 Sending a GossipDigestAckMessage to /127.0.0.12
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:4
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,904 Received a GossipDigestSynMessage from /127.0.0.1
TRACE 15:44:07,904 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,904 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,905 @@@@ Size of GossipDigestAckMessage is 136
&&&Added socket : 57497
TRACE 15:44:07,905 Sending a GossipDigestAckMessage to /127.0.0.1
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:5
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,906 Received a GossipDigestSynMessage from /127.0.0.12
TRACE 15:44:07,906 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,906 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
&&&Added socket : 59078
TRACE 15:44:07,907 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,907 Sending a GossipDigestAckMessage to /127.0.0.12
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:6
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,907 Received a GossipDigestSynMessage from /127.0.0.13
TRACE 15:44:07,908 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,908 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,908 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,909 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:7
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,909 Received a GossipDigestSynMessage from /127.0.0.12
TRACE 15:44:07,910 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,910 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,910 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,910 Sending a GossipDigestAckMessage to /127.0.0.12
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:8
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,911 Received a GossipDigestSynMessage from /127.0.0.12
TRACE 15:44:07,911 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,911 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,912 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,912 Sending a GossipDigestAckMessage to /127.0.0.12
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:9
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,912 Received a GossipDigestSynMessage from /127.0.0.13
TRACE 15:44:07,913 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,913 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,913 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,914 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:10
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,914 Received a GossipDigestSynMessage from /127.0.0.13
DEBUG 15:44:07,915 attempting to connect to /127.0.0.1
&&&Added socket : 50851
TRACE 15:44:07,936 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,937 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,937 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,941 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:11
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,942 Received a GossipDigestSynMessage from /127.0.0.13
TRACE 15:44:07,942 Scanning for state greater than 0 for /127.0.0.11
TRACE 15:44:07,942 Adding state MOVE: NORMAL,115334859289675963285972157264705071312
TRACE 15:44:07,943 @@@@ Size of GossipDigestAckMessage is 136
TRACE 15:44:07,946 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:12
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:07,948 Received a GossipDigestAck2Message from /127.0.0.12
 INFO 15:44:07,951 Node /127.0.0.12 is now part of the cluster
DEBUG 15:44:07,952 Resetting pool for /127.0.0.12
DEBUG 15:44:07,958 Node /127.0.0.12 state normal, token 95763128674829263755597563476114486239
- TM uNT(1) : size of sortedToken = 1
- TM uNT(2) : size of sortedToken = 2 ... leavingEndPoints = []
TokenSizeTest:2:-1009577
DEBUG 15:44:07,959 No bootstrapping or leaving nodes -> empty pending ranges for Keyspace1
 INFO 15:44:07,959 Node /127.0.0.13 is now part of the cluster
DEBUG 15:44:07,960 Resetting pool for /127.0.0.13
DEBUG 15:44:07,960 Node /127.0.0.13 state normal, token 20902772761227803006582482381914476626
- TM uNT(1) : size of sortedToken = 2
- TM uNT(2) : size of sortedToken = 3 ... leavingEndPoints = []
TokenSizeTest:3:-1009577
DEBUG 15:44:07,961 No bootstrapping or leaving nodes -> empty pending ranges for Keyspace1
 INFO 15:44:07,962 Node /127.0.0.1 is now part of the cluster
DEBUG 15:44:07,962 Resetting pool for /127.0.0.1
DEBUG 15:44:07,962 Node /127.0.0.1 state normal, token 151364447406687126714367278325169556726
- TM uNT(1) : size of sortedToken = 3
- TM uNT(2) : size of sortedToken = 4 ... leavingEndPoints = []
TokenSizeTest:4:-1009577
DEBUG 15:44:07,963 No bootstrapping or leaving nodes -> empty pending ranges for Keyspace1
TRACE 15:44:07,964 Received a GossipDigestAck2Message from /127.0.0.1
TRACE 15:44:07,964 reporting /127.0.0.13
TRACE 15:44:07,966 reporting /127.0.0.1
TRACE 15:44:07,967 marking as alive /127.0.0.13
 INFO 15:44:07,970 InetAddress /127.0.0.13 is now UP
 INFO 15:44:07,970 Started hinted handoff for endPoint /127.0.0.13
 INFO 15:44:07,970 /127.0.0.13is cassReady
TRACE 15:44:07,971 Updating heartbeat state version to 31 from 30 for /127.0.0.13 ...
TRACE 15:44:07,972 marking as alive /127.0.0.1
 INFO 15:44:07,972 InetAddress /127.0.0.1 is now UP
 INFO 15:44:07,972 /127.0.0.1is cassReady
TRACE 15:44:07,972 Updating heartbeat state version to 128 from 126 for /127.0.0.1 ...
TRACE 15:44:07,973 Received a GossipDigestAck2Message from /127.0.0.1
TRACE 15:44:07,973 Received a GossipDigestAck2Message from /127.0.0.12
TRACE 15:44:07,974 Received a GossipDigestAck2Message from /127.0.0.12
TRACE 15:44:07,974 Received a GossipDigestAck2Message from /127.0.0.12
TRACE 15:44:07,975 Received a GossipDigestAck2Message from /127.0.0.12
TRACE 15:44:07,975 Received a GossipDigestAck2Message from /127.0.0.13
TRACE 15:44:07,976 Received a GossipDigestAck2Message from /127.0.0.13
TRACE 15:44:07,976 Received a GossipDigestAck2Message from /127.0.0.13
TRACE 15:44:07,977 Received a GossipDigestAck2Message from /127.0.0.13
TRACE 15:44:07,978 Received a GossipDigestAck2Message from /127.0.0.13
 INFO 15:44:07,991 Finished hinted handoff of 0 rows to endpoint /127.0.0.13
 INFO 15:44:07,992 Started hinted handoff for endPoint /127.0.0.1
 INFO 15:44:07,993 Finished hinted handoff of 0 rows to endpoint /127.0.0.1
TRACE 15:44:08,060 Received a GossipDigestSynMessage from /127.0.0.12
TRACE 15:44:08,061 reporting /127.0.0.12
TRACE 15:44:08,061 reporting /127.0.0.12
TRACE 15:44:08,062 Scanning for state greater than 30 for /127.0.0.13
TRACE 15:44:08,063 Scanning for state greater than 126 for /127.0.0.1
TRACE 15:44:08,063 @@@@ Size of GossipDigestAckMessage is 69
TRACE 15:44:08,064 Sending a GossipDigestAckMessage to /127.0.0.12
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:13
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

DEBUG 15:44:08,064 attempting to connect to /127.0.0.12
&&&Added socket : 51856
TRACE 15:44:08,070 Received a GossipDigestAck2Message from /127.0.0.12
TRACE 15:44:08,071 reporting /127.0.0.12
TRACE 15:44:08,071 marking as alive /127.0.0.12
 INFO 15:44:08,071 InetAddress /127.0.0.12 is now UP
 INFO 15:44:08,072 /127.0.0.12is cassReady
 INFO 15:44:08,071 Started hinted handoff for endPoint /127.0.0.12
TRACE 15:44:08,072 Updating heartbeat state version to 17 from 16 for /127.0.0.12 ...
 INFO 15:44:08,073 Finished hinted handoff of 0 rows to endpoint /127.0.0.12
TRACE 15:44:08,260 Received a GossipDigestSynMessage from /127.0.0.13
TRACE 15:44:08,261 reporting /127.0.0.13
TRACE 15:44:08,261 reporting /127.0.0.13
TRACE 15:44:08,262 Scanning for state greater than 15 for /127.0.0.12
TRACE 15:44:08,263 @@@@ Size of GossipDigestAckMessage is 52
TRACE 15:44:08,263 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:14
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

DEBUG 15:44:08,264 attempting to connect to /127.0.0.13
&&&Added socket : 48341
TRACE 15:44:08,274 Received a GossipDigestAck2Message from /127.0.0.13
TRACE 15:44:08,275 reporting /127.0.0.13
TRACE 15:44:08,275 marking as alive /127.0.0.13
TRACE 15:44:08,275 Updating heartbeat state version to 32 from 31 for /127.0.0.13 ...
TRACE 15:44:08,342 Received a GossipDigestSynMessage from /127.0.0.1
TRACE 15:44:08,342 reporting /127.0.0.1
TRACE 15:44:08,343 reporting /127.0.0.1
TRACE 15:44:08,344 Scanning for state greater than 15 for /127.0.0.12
TRACE 15:44:08,344 Scanning for state greater than 31 for /127.0.0.13
TRACE 15:44:08,345 @@@@ Size of GossipDigestAckMessage is 69
TRACE 15:44:08,346 Sending a GossipDigestAckMessage to /127.0.0.1
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:15
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

DEBUG 15:44:08,347 attempting to connect to /127.0.0.1
&&&Added socket : 35902
TRACE 15:44:08,355 Received a GossipDigestAck2Message from /127.0.0.1
TRACE 15:44:08,356 reporting /127.0.0.1
TRACE 15:44:08,357 marking as alive /127.0.0.1
TRACE 15:44:08,357 Updating heartbeat state version to 129 from 128 for /127.0.0.1 ...
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:08,745 Gossip Digests are : /127.0.0.11:1308955447:2 /127.0.0.1:1308955160:129 /127.0.0.12:1308955413:17 /127.0.0.11:1308955447:2 /127.0.0.13:1308955372:32 
TRACE 15:44:08,748 Sending a GossipDigestSynMessage to /127.0.0.1 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:16
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:08,750 Performing status check ...
TRACE 15:44:08,753 Received a GossipDigestAckMessage from /127.0.0.1
DEBUG 15:44:08,755 GC for ParNew: 42 ms, 30357480 reclaimed leaving 44987536 used; max is 654114816
TRACE 15:44:08,754 PHI for /127.0.0.13 : 1.0272941078106548
TRACE 15:44:08,756 PHI for /127.0.0.1 : 0.7768503541460413
TRACE 15:44:08,756 PHI for /127.0.0.12 : 1.7499512947278093
DEBUG 15:44:08,757 GC for ConcurrentMarkSweep: -3077252 ms, 0 reclaimed leaving 0 used; max is 0
TRACE 15:44:08,757 Scanning for state greater than 1 for /127.0.0.11
TRACE 15:44:08,757 Scanning for state greater than 1 for /127.0.0.11
TRACE 15:44:08,757 Sending a GossipDigestAck2Message to /127.0.0.1
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:17
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:09,059 Received a GossipDigestSynMessage from /127.0.0.12
TRACE 15:44:09,060 reporting /127.0.0.12
TRACE 15:44:09,060 reporting /127.0.0.12
TRACE 15:44:09,061 Scanning for state greater than 1 for /127.0.0.11
TRACE 15:44:09,061 Scanning for state greater than 31 for /127.0.0.13
TRACE 15:44:09,061 Scanning for state greater than 128 for /127.0.0.1
TRACE 15:44:09,061 @@@@ Size of GossipDigestAckMessage is 86
TRACE 15:44:09,062 Sending a GossipDigestAckMessage to /127.0.0.12
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:18
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:09,065 Received a GossipDigestAck2Message from /127.0.0.12
TRACE 15:44:09,066 reporting /127.0.0.12
TRACE 15:44:09,066 marking as alive /127.0.0.12
TRACE 15:44:09,066 Updating heartbeat state version to 18 from 17 for /127.0.0.12 ...
node1 recorded Tokensize 4
- Nodes connected, Gossiper returning 
DEBUG 15:44:09,762 Disseminating load info ...
DEBUG 15:44:10,330 Applying RowMutation(table='Keyspace1', key='key-014', modifications=[ColumnFamily(Standard1 [6578706572696d656e7456616c7565:false:8@1308955450321,])])
DEBUG 15:44:10,338 RowMutation(table='Keyspace1', key='key-014', modifications=[ColumnFamily(Standard1 [6578706572696d656e7456616c7565:false:8@1308955450321,])]) applied.  Sending response to 453@/127.0.0.1
Sending message ::
-->ID:453
FROM:/127.0.0.11
TYPE:RESPONSE-STAGE
VERB:READ_RESPONSE

node1 recorded Tokensize 4 [part of workload]
- Nodes connected, Gossiper returning  [part of workload]

## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:AFTER / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(59)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:59)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(60)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:60)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:AFTER / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(60)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:60)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(71)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:71)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:AFTER / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(71)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:71)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(void java.io.DataInputStream.readFully(byte[]))
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(73)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:73)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(void java.io.DataInputStream.readFully(byte[]))
   ** PL:AFTER / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(73)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:73)


DEBUG 15:44:10,832 Read key key-014; sending response to D852BFFE-D301-0BB4-4857-1C123ECA6A48@/127.0.0.1

## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(59)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:59)


wRRPATH ReadVerbHanlder doVerb(): Read key key-014; sending response to D852BFFE-D301-0BB4-4857-1C123ECA6A48@/127.0.0.1 [part of workload]
    [00] java.lang.Thread.getStackTrace(Thread.java:1436) 
    [01] org.apache.cassandra.Util.getStackTrace(Util.java:136) 
    [02] org.apache.cassandra.Util.printStackTrace(Util.java:123) 
    [03] org.apache.cassandra.Util.printStackTrace(Util.java:130) 
    [04] org.apache.cassandra.db.ReadVerbHandler.doVerb(ReadVerbHandler.java:100) 
    [05] org.apache.cassandra.net.MessageDeliveryTask.run(MessageDeliveryTask.java:49) 
    [06] java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886) 
    [07] java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908) 
    [08] java.lang.Thread.run(Thread.java:619) 
cutSendOneWay targetIO : NetIO-Node0:127.0.0.11
corrupt :: failType returned :: NONE
Sending message ::
-->ID:D852BFFE-D301-0BB4-4857-1C123ECA6A48
FROM:/127.0.0.11
TYPE:RESPONSE-STAGE
VERB:READ_RESPONSE
 [part of workload]

## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node1:127.0.0.1:7000] at *Node1* :: [Undecided] message 

  call(void java.io.DataOutputStream.write(byte[], int, int))
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:WRITE **
    SourceLoc: org/apache/cassandra/net/OutboundTcpConnection.java(101)

      [0] org.apache.cassandra.net.OutboundTcpConnection (writeConnected:101)
      [1] org.apache.cassandra.net.OutboundTcpConnection (run:90)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node1:127.0.0.1:7000] at *Node1* :: [Undecided] message 

  call(void java.io.DataOutputStream.write(byte[], int, int))
   ** PL:AFTER / EX:IO / BOOL:NO / IO:WRITE **
    SourceLoc: org/apache/cassandra/net/OutboundTcpConnection.java(101)

      [0] org.apache.cassandra.net.OutboundTcpConnection (writeConnected:101)
      [1] org.apache.cassandra.net.OutboundTcpConnection (run:90)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node1:127.0.0.1:7000] at *Node1* :: [Undecided] message 

  call(void java.io.DataOutputStream.flush())
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:WRITE **
    SourceLoc: org/apache/cassandra/net/OutboundTcpConnection.java(104)

      [0] org.apache.cassandra.net.OutboundTcpConnection (writeConnected:104)
      [1] org.apache.cassandra.net.OutboundTcpConnection (run:90)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node1:127.0.0.1:7000] at *Node1* :: [Undecided] message 

  call(void java.io.DataOutputStream.flush())
   ** PL:AFTER / EX:IO / BOOL:NO / IO:WRITE **
    SourceLoc: org/apache/cassandra/net/OutboundTcpConnection.java(104)

      [0] org.apache.cassandra.net.OutboundTcpConnection (writeConnected:104)
      [1] org.apache.cassandra.net.OutboundTcpConnection (run:90)


node1 recorded Tokensize 4 [part of workload]
- Nodes connected, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Nodes connected, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Nodes connected, Gossiper returning  [part of workload]
DEBUG 15:44:13,945 Applying RowMutation(table='Keyspace1', key='key-014', modifications=[ColumnFamily(Standard1 [6578706572696d656e7456616c7565:true:4@1308955453939,])])
DEBUG 15:44:13,951 RowMutation(table='Keyspace1', key='key-014', modifications=[ColumnFamily(Standard1 [6578706572696d656e7456616c7565:true:4@1308955453939,])]) applied.  Sending response to 456@/127.0.0.1
Sending message ::
-->ID:456
FROM:/127.0.0.11
TYPE:RESPONSE-STAGE
VERB:READ_RESPONSE

node1 recorded Tokensize 4
- Nodes connected, Gossiper returning 
DEBUG 15:44:15,179 Applying RowMutation(table='Keyspace1', key='key-015', modifications=[ColumnFamily(Standard1 [6578706572696d656e7456616c7565:false:8@1308955455172,])])
DEBUG 15:44:15,179 RowMutation(table='Keyspace1', key='key-015', modifications=[ColumnFamily(Standard1 [6578706572696d656e7456616c7565:false:8@1308955455172,])]) applied.  Sending response to 457@/127.0.0.1
Sending message ::
-->ID:457
FROM:/127.0.0.11
TYPE:RESPONSE-STAGE
VERB:READ_RESPONSE


## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:AFTER / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(59)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:59)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(60)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:60)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:AFTER / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(60)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:60)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(71)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:71)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:AFTER / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(71)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:71)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(void java.io.DataInputStream.readFully(byte[]))
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(73)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:73)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(void java.io.DataInputStream.readFully(byte[]))
   ** PL:AFTER / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(73)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:73)


DEBUG 15:44:15,380 Read key key-015; sending response to 48B3AFAD-697D-6A1B-98FA-1632BFEFB543@/127.0.0.1
wRRPATH ReadVerbHanlder doVerb(): Read key key-015; sending response to 48B3AFAD-697D-6A1B-98FA-1632BFEFB543@/127.0.0.1 [part of workload]
    [00] java.lang.Thread.getStackTrace(Thread.java:1436) 
    [01] org.apache.cassandra.Util.getStackTrace(Util.java:136) 
    [02] org.apache.cassandra.Util.printStackTrace(Util.java:123) 
    [03] org.apache.cassandra.Util.printStackTrace(Util.java:130) 
    [04] org.apache.cassandra.db.ReadVerbHandler.doVerb(ReadVerbHandler.java:100) 
    [05] org.apache.cassandra.net.MessageDeliveryTask.run(MessageDeliveryTask.java:49) 
    [06] java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886) 
    [07] java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908) 
    [08] java.lang.Thread.run(Thread.java:619) 
cutSendOneWay targetIO : NetIO-Node0:127.0.0.11
corrupt :: failType returned :: NONE
Sending message ::
-->ID:48B3AFAD-697D-6A1B-98FA-1632BFEFB543
FROM:/127.0.0.11
TYPE:RESPONSE-STAGE
VERB:READ_RESPONSE
 [part of workload]

## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node1:127.0.0.1:7000] at *Node1* :: [Undecided] message 

  call(void java.io.DataOutputStream.write(byte[], int, int))
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:WRITE **
    SourceLoc: org/apache/cassandra/net/OutboundTcpConnection.java(101)

      [0] org.apache.cassandra.net.OutboundTcpConnection (writeConnected:101)
      [1] org.apache.cassandra.net.OutboundTcpConnection (run:90)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node1:127.0.0.1:7000] at *Node1* :: [Undecided] message 

  call(void java.io.DataOutputStream.write(byte[], int, int))
   ** PL:AFTER / EX:IO / BOOL:NO / IO:WRITE **
    SourceLoc: org/apache/cassandra/net/OutboundTcpConnection.java(101)

      [0] org.apache.cassandra.net.OutboundTcpConnection (writeConnected:101)
      [1] org.apache.cassandra.net.OutboundTcpConnection (run:90)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node1:127.0.0.1:7000] at *Node1* :: [Undecided] message 

  call(void java.io.DataOutputStream.flush())
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:WRITE **
    SourceLoc: org/apache/cassandra/net/OutboundTcpConnection.java(104)

      [0] org.apache.cassandra.net.OutboundTcpConnection (writeConnected:104)
      [1] org.apache.cassandra.net.OutboundTcpConnection (run:90)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node0:127.0.0.1:47901] at *Node1* :: [Undecided] message 

  call(int java.io.DataInputStream.readInt())
   ** PL:BEFORE / EX:IO / BOOL:NO / IO:READ **
    SourceLoc: org/apache/cassandra/net/IncomingTcpConnection.java(59)

      [0] org.apache.cassandra.net.IncomingTcpConnection (run:59)



## ############################################
## MESSAGE: I'm failing this (see below) with FailType: NONE
## ############################################

  [targetIO][NetIO-Node1:127.0.0.1:7000] at *Node1* :: [Undecided] message 

  call(void java.io.DataOutputStream.flush())
   ** PL:AFTER / EX:IO / BOOL:NO / IO:WRITE **
    SourceLoc: org/apache/cassandra/net/OutboundTcpConnection.java(104)

      [0] org.apache.cassandra.net.OutboundTcpConnection (writeConnected:104)
      [1] org.apache.cassandra.net.OutboundTcpConnection (run:90)


TRACE 15:44:15,737 eof reading from socket; closing
java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:375)
	at org.apache.cassandra.net.IncomingTcpConnection.readInt_aroundBody4(IncomingTcpConnection.java:59)
	at org.apache.cassandra.net.IncomingTcpConnection.readInt_aroundBody5$advice(IncomingTcpConnection.java:236)
	at org.apache.cassandra.net.IncomingTcpConnection.run(IncomingTcpConnection.java:59)
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
node1 recorded Tokensize 4 [part of workload]
- Experiment running, Gossiper returning  [part of workload]
DEBUG 15:44:28,205 Applying RowMutation(table='Keyspace1', key='key-015', modifications=[ColumnFamily(Standard1 [6578706572696d656e7456616c7565:true:4@1308955468199,])])
DEBUG 15:44:28,206 RowMutation(table='Keyspace1', key='key-015', modifications=[ColumnFamily(Standard1 [6578706572696d656e7456616c7565:true:4@1308955468199,])]) applied.  Sending response to 460@/127.0.0.1
Sending message ::
-->ID:460
FROM:/127.0.0.11
TYPE:RESPONSE-STAGE
VERB:READ_RESPONSE

node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:28,747 Gossip Digests are : /127.0.0.11:1308955447:4 /127.0.0.1:1308955160:129 /127.0.0.11:1308955447:4 /127.0.0.13:1308955372:32 /127.0.0.12:1308955413:18 
TRACE 15:44:28,748 Sending a GossipDigestSynMessage to /127.0.0.13 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:19
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:28,749 Performing status check ...
TRACE 15:44:28,750 PHI for /127.0.0.13 : Infinity
 INFO 15:44:28,750 InetAddress /127.0.0.13 is now dead.
DEBUG 15:44:28,751 Resetting pool for /127.0.0.13
TRACE 15:44:28,752 PHI for /127.0.0.1 : Infinity
 INFO 15:44:28,753 InetAddress /127.0.0.1 is now dead.
TRACE 15:44:28,755 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:28,756 reporting /127.0.0.13
DEBUG 15:44:28,756 Resetting pool for /127.0.0.1
TRACE 15:44:28,757 PHI for /127.0.0.12 : Infinity
 INFO 15:44:28,758 InetAddress /127.0.0.12 is now dead.
DEBUG 15:44:28,758 Resetting pool for /127.0.0.12
TRACE 15:44:28,759 marking as alive /127.0.0.13
 INFO 15:44:28,760 InetAddress /127.0.0.13 is now UP
 INFO 15:44:28,760 /127.0.0.13is cassReady
 INFO 15:44:28,760 Started hinted handoff for endPoint /127.0.0.13
TRACE 15:44:28,761 Updating heartbeat state version to 33 from 32 for /127.0.0.13 ...
TRACE 15:44:28,762 Scanning for state greater than 1 for /127.0.0.11
 INFO 15:44:28,762 Finished hinted handoff of 0 rows to endpoint /127.0.0.13
TRACE 15:44:28,762 Adding state LOAD-INFORMATION: 495.0
TRACE 15:44:28,765 Scanning for state greater than 1 for /127.0.0.11
TRACE 15:44:28,765 Adding state LOAD-INFORMATION: 495.0
TRACE 15:44:28,766 Scanning for state greater than 17 for /127.0.0.12
TRACE 15:44:28,766 Scanning for state greater than 128 for /127.0.0.1
TRACE 15:44:28,767 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:20
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

DEBUG 15:44:28,768 attempting to connect to /127.0.0.13
&&&Added socket : 33966
TRACE 15:44:29,345 Received a GossipDigestSynMessage from /127.0.0.1
TRACE 15:44:29,346 reporting /127.0.0.1
TRACE 15:44:29,347 reporting /127.0.0.1
TRACE 15:44:29,348 Scanning for state greater than 2 for /127.0.0.11
TRACE 15:44:29,348 Adding state LOAD-INFORMATION: 495.0
TRACE 15:44:29,349 Scanning for state greater than 17 for /127.0.0.12
TRACE 15:44:29,349 Scanning for state greater than 32 for /127.0.0.13
TRACE 15:44:29,350 @@@@ Size of GossipDigestAckMessage is 115
TRACE 15:44:29,351 Sending a GossipDigestAckMessage to /127.0.0.1
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:21
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

DEBUG 15:44:29,352 attempting to connect to /127.0.0.1
&&&Added socket : 56767
TRACE 15:44:29,361 Received a GossipDigestAck2Message from /127.0.0.1
TRACE 15:44:29,362 reporting /127.0.0.1
TRACE 15:44:29,362 marking as alive /127.0.0.1
 INFO 15:44:29,363 InetAddress /127.0.0.1 is now UP
 INFO 15:44:29,363 /127.0.0.1is cassReady
 INFO 15:44:29,363 Started hinted handoff for endPoint /127.0.0.1
TRACE 15:44:29,364 Updating heartbeat state version to 132 from 129 for /127.0.0.1 ...
 INFO 15:44:29,365 Finished hinted handoff of 0 rows to endpoint /127.0.0.1
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:29,747 Gossip Digests are : /127.0.0.11:1308955447:5 /127.0.0.1:1308955160:132 /127.0.0.13:1308955372:33 /127.0.0.12:1308955413:18 /127.0.0.11:1308955447:5 
TRACE 15:44:29,748 Sending a GossipDigestSynMessage to /127.0.0.1 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:22
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:29,749 Sending a GossipDigestSynMessage to /127.0.0.1 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:22
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:29,750 Performing status check ...
TRACE 15:44:29,750 PHI for /127.0.0.13 : 0.10137821497624168
TRACE 15:44:29,753 Received a GossipDigestAckMessage from /127.0.0.1
TRACE 15:44:29,755 PHI for /127.0.0.1 : 0.05456698422999983
TRACE 15:44:29,755 PHI for /127.0.0.12 : Infinity
TRACE 15:44:29,756 Scanning for state greater than 4 for /127.0.0.11
TRACE 15:44:29,757 Scanning for state greater than 4 for /127.0.0.11
TRACE 15:44:29,757 Sending a GossipDigestAck2Message to /127.0.0.1
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:23
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:29,758 Received a GossipDigestAckMessage from /127.0.0.1
TRACE 15:44:29,759 Scanning for state greater than 4 for /127.0.0.11
TRACE 15:44:29,759 Scanning for state greater than 4 for /127.0.0.11
TRACE 15:44:29,760 Sending a GossipDigestAck2Message to /127.0.0.1
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:24
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:30,263 Received a GossipDigestSynMessage from /127.0.0.13
TRACE 15:44:30,264 reporting /127.0.0.13
TRACE 15:44:30,264 reporting /127.0.0.13
TRACE 15:44:30,265 Scanning for state greater than 4 for /127.0.0.11
TRACE 15:44:30,266 Scanning for state greater than 129 for /127.0.0.1
TRACE 15:44:30,279 Adding state LOAD-INFORMATION: 495.0
TRACE 15:44:30,279 @@@@ Size of GossipDigestAckMessage is 98
TRACE 15:44:30,279 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:25
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:30,282 Received a GossipDigestAck2Message from /127.0.0.13
TRACE 15:44:30,283 reporting /127.0.0.13
TRACE 15:44:30,283 marking as alive /127.0.0.13
TRACE 15:44:30,283 Updating heartbeat state version to 35 from 33 for /127.0.0.13 ...
TRACE 15:44:30,345 Received a GossipDigestSynMessage from /127.0.0.1
TRACE 15:44:30,346 reporting /127.0.0.1
TRACE 15:44:30,347 reporting /127.0.0.1
TRACE 15:44:30,348 Scanning for state greater than 33 for /127.0.0.13
TRACE 15:44:30,349 @@@@ Size of GossipDigestAckMessage is 52
TRACE 15:44:30,349 Sending a GossipDigestAckMessage to /127.0.0.1
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:26
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:30,354 Received a GossipDigestAck2Message from /127.0.0.1
TRACE 15:44:30,355 reporting /127.0.0.1
TRACE 15:44:30,355 marking as alive /127.0.0.1
TRACE 15:44:30,356 Updating heartbeat state version to 133 from 132 for /127.0.0.1 ...
DEBUG 15:44:30,738 GC for ParNew: 12 ms, 32806208 reclaimed leaving 47254256 used; max is 654114816
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:30,747 Gossip Digests are : /127.0.0.11:1308955447:6 /127.0.0.12:1308955413:18 /127.0.0.13:1308955372:35 /127.0.0.1:1308955160:133 /127.0.0.11:1308955447:6 
TRACE 15:44:30,748 Sending a GossipDigestSynMessage to /127.0.0.13 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:27
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:30,749 Sending a GossipDigestSynMessage to /127.0.0.12 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:27
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:30,750 Sending a GossipDigestSynMessage to /127.0.0.13 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:27
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

DEBUG 15:44:30,752 attempting to connect to /127.0.0.12
TRACE 15:44:30,753 Performing status check ...
TRACE 15:44:30,754 PHI for /127.0.0.13 : 0.07171625943603528
TRACE 15:44:30,754 PHI for /127.0.0.1 : 0.07570932291130611
TRACE 15:44:30,755 PHI for /127.0.0.12 : Infinity
TRACE 15:44:30,757 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:30,758 Scanning for state greater than 5 for /127.0.0.11
TRACE 15:44:30,758 Scanning for state greater than 5 for /127.0.0.11
TRACE 15:44:30,758 Scanning for state greater than 132 for /127.0.0.1
TRACE 15:44:30,759 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:28
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:30,759 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:30,759 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:30,760 Scanning for state greater than 5 for /127.0.0.11
TRACE 15:44:30,760 Scanning for state greater than 5 for /127.0.0.11
TRACE 15:44:30,760 Scanning for state greater than 132 for /127.0.0.1
TRACE 15:44:30,761 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:29
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:30,863 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:30,967 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:31,071 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:31,175 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:31,279 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:31,346 Received a GossipDigestSynMessage from /127.0.0.1
TRACE 15:44:31,346 reporting /127.0.0.1
TRACE 15:44:31,347 reporting /127.0.0.1
TRACE 15:44:31,348 reporting /127.0.0.13
TRACE 15:44:31,349 @@@@ Size of GossipDigestAckMessage is 48
TRACE 15:44:31,349 Sending a GossipDigestAckMessage to /127.0.0.1
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:30
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:31,354 Received a GossipDigestAck2Message from /127.0.0.1
TRACE 15:44:31,355 reporting /127.0.0.1
TRACE 15:44:31,355 reporting /127.0.0.13
TRACE 15:44:31,356 marking as alive /127.0.0.1
TRACE 15:44:31,356 Updating heartbeat state version to 134 from 133 for /127.0.0.1 ...
TRACE 15:44:31,357 marking as alive /127.0.0.13
TRACE 15:44:31,357 Updating heartbeat state version to 36 from 35 for /127.0.0.13 ...
TRACE 15:44:31,382 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:31,486 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:31,589 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:31,693 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:31,747 Gossip Digests are : /127.0.0.11:1308955447:7 /127.0.0.12:1308955413:18 /127.0.0.1:1308955160:134 /127.0.0.13:1308955372:36 /127.0.0.11:1308955447:7 
TRACE 15:44:31,748 Sending a GossipDigestSynMessage to /127.0.0.13 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:31
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:31,748 Sending a GossipDigestSynMessage to /127.0.0.12 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:31
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:31,752 Performing status check ...
TRACE 15:44:31,753 PHI for /127.0.0.13 : 0.07216730539349167
TRACE 15:44:31,753 PHI for /127.0.0.1 : 0.09406562497351915
TRACE 15:44:31,753 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:31,753 PHI for /127.0.0.12 : Infinity
TRACE 15:44:31,754 Scanning for state greater than 6 for /127.0.0.11
TRACE 15:44:31,754 Scanning for state greater than 6 for /127.0.0.11
TRACE 15:44:31,754 Scanning for state greater than 133 for /127.0.0.1
TRACE 15:44:31,754 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:32
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:31,797 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:31,900 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,004 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,108 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,212 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,316 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,422 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,526 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,630 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,734 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:32,747 Gossip Digests are : /127.0.0.11:1308955447:8 /127.0.0.13:1308955372:36 /127.0.0.11:1308955447:8 /127.0.0.1:1308955160:134 /127.0.0.12:1308955413:18 
TRACE 15:44:32,748 Sending a GossipDigestSynMessage to /127.0.0.13 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:33
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:32,748 Sending a GossipDigestSynMessage to /127.0.0.12 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:33
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:32,749 Sending a GossipDigestSynMessage to /127.0.0.13 ...
TRACE 15:44:32,754 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:32,755 reporting /127.0.0.13
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:33
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:32,760 Performing status check ...
TRACE 15:44:32,762 PHI for /127.0.0.13 : 9.444917558196563E-4
TRACE 15:44:32,762 PHI for /127.0.0.1 : 0.3325385284867875
TRACE 15:44:32,763 PHI for /127.0.0.12 : Infinity
TRACE 15:44:32,763 marking as alive /127.0.0.13
TRACE 15:44:32,764 Updating heartbeat state version to 37 from 36 for /127.0.0.13 ...
TRACE 15:44:32,764 Scanning for state greater than 7 for /127.0.0.11
TRACE 15:44:32,765 Scanning for state greater than 7 for /127.0.0.11
TRACE 15:44:32,765 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:34
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:32,769 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:32,769 Scanning for state greater than 7 for /127.0.0.11
TRACE 15:44:32,769 Scanning for state greater than 7 for /127.0.0.11
TRACE 15:44:32,769 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:35
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:32,838 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:32,943 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,047 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,151 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,255 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,262 Received a GossipDigestSynMessage from /127.0.0.13
TRACE 15:44:33,263 reporting /127.0.0.13
TRACE 15:44:33,263 reporting /127.0.0.13
TRACE 15:44:33,264 @@@@ Size of GossipDigestAckMessage is 35
TRACE 15:44:33,265 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:36
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:33,270 Received a GossipDigestAck2Message from /127.0.0.13
TRACE 15:44:33,271 reporting /127.0.0.13
TRACE 15:44:33,271 marking as alive /127.0.0.13
TRACE 15:44:33,272 Updating heartbeat state version to 38 from 37 for /127.0.0.13 ...
TRACE 15:44:33,359 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,463 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,568 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,672 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:33,747 Gossip Digests are : /127.0.0.11:1308955447:9 /127.0.0.12:1308955413:18 /127.0.0.1:1308955160:134 /127.0.0.11:1308955447:9 /127.0.0.13:1308955372:38 
TRACE 15:44:33,748 Sending a GossipDigestSynMessage to /127.0.0.1 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:37
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:33,749 Sending a GossipDigestSynMessage to /127.0.0.12 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:37
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:33,749 Sending a GossipDigestSynMessage to /127.0.0.13 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:37
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:33,754 Received a GossipDigestAckMessage from /127.0.0.1
TRACE 15:44:33,754 reporting /127.0.0.1
TRACE 15:44:33,755 Performing status check ...
TRACE 15:44:33,755 PHI for /127.0.0.13 : 0.11403469772054695
TRACE 15:44:33,756 PHI for /127.0.0.1 : 2.3128890545668363E-4
TRACE 15:44:33,756 PHI for /127.0.0.12 : Infinity
TRACE 15:44:33,758 marking as alive /127.0.0.1
TRACE 15:44:33,759 Updating heartbeat state version to 136 from 134 for /127.0.0.1 ...
TRACE 15:44:33,759 Scanning for state greater than 6 for /127.0.0.11
TRACE 15:44:33,760 Scanning for state greater than 6 for /127.0.0.11
TRACE 15:44:33,760 Scanning for state greater than 36 for /127.0.0.13
TRACE 15:44:33,760 Sending a GossipDigestAck2Message to /127.0.0.1
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:38
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:33,761 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:33,761 Scanning for state greater than 8 for /127.0.0.11
TRACE 15:44:33,761 Scanning for state greater than 8 for /127.0.0.11
TRACE 15:44:33,761 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:39
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:33,775 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,879 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:33,984 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:34,088 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:34,192 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:34,296 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:34,346 Received a GossipDigestSynMessage from /127.0.0.1
TRACE 15:44:34,347 reporting /127.0.0.1
TRACE 15:44:34,347 reporting /127.0.0.1
TRACE 15:44:34,348 reporting /127.0.0.13
TRACE 15:44:34,349 @@@@ Size of GossipDigestAckMessage is 48
TRACE 15:44:34,349 Sending a GossipDigestAckMessage to /127.0.0.1
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:40
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:34,354 Received a GossipDigestAck2Message from /127.0.0.1
TRACE 15:44:34,354 reporting /127.0.0.1
TRACE 15:44:34,355 reporting /127.0.0.13
TRACE 15:44:34,355 marking as alive /127.0.0.1
TRACE 15:44:34,356 Updating heartbeat state version to 137 from 136 for /127.0.0.1 ...
TRACE 15:44:34,356 marking as alive /127.0.0.13
TRACE 15:44:34,357 Updating heartbeat state version to 39 from 38 for /127.0.0.13 ...
TRACE 15:44:34,400 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:34,504 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:34,608 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:34,712 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:34,747 Gossip Digests are : /127.0.0.11:1308955447:10 /127.0.0.11:1308955447:10 /127.0.0.12:1308955413:18 /127.0.0.13:1308955372:39 /127.0.0.1:1308955160:137 
TRACE 15:44:34,748 Sending a GossipDigestSynMessage to /127.0.0.1 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:41
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:34,749 Sending a GossipDigestSynMessage to /127.0.0.13 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:41
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:34,752 Performing status check ...
TRACE 15:44:34,752 PHI for /127.0.0.13 : 0.10258975637967481
TRACE 15:44:34,753 Received a GossipDigestAckMessage from /127.0.0.1
TRACE 15:44:34,754 PHI for /127.0.0.1 : 0.10955889135487046
TRACE 15:44:34,755 PHI for /127.0.0.12 : Infinity
TRACE 15:44:34,755 Scanning for state greater than 9 for /127.0.0.11
TRACE 15:44:34,756 Scanning for state greater than 9 for /127.0.0.11
TRACE 15:44:34,757 Sending a GossipDigestAck2Message to /127.0.0.1
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:42
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:34,758 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:34,758 Scanning for state greater than 9 for /127.0.0.11
TRACE 15:44:34,759 Scanning for state greater than 9 for /127.0.0.11
TRACE 15:44:34,759 Scanning for state greater than 136 for /127.0.0.1
TRACE 15:44:34,760 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:43
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:34,815 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:34,918 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,020 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,124 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,227 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,331 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,435 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,539 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,644 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:35,747 Gossip Digests are : /127.0.0.11:1308955447:11 /127.0.0.1:1308955160:137 /127.0.0.11:1308955447:11 /127.0.0.13:1308955372:39 /127.0.0.12:1308955413:18 
TRACE 15:44:35,748 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,748 Sending a GossipDigestSynMessage to /127.0.0.1 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:44
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:35,753 Sending a GossipDigestSynMessage to /127.0.0.1 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:44
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:35,754 Received a GossipDigestAckMessage from /127.0.0.1
TRACE 15:44:35,755 reporting /127.0.0.1
TRACE 15:44:35,756 Performing status check ...
TRACE 15:44:35,757 PHI for /127.0.0.13 : 0.36229430338615637
TRACE 15:44:35,757 PHI for /127.0.0.1 : 5.526937693904278E-4
TRACE 15:44:35,765 PHI for /127.0.0.12 : Infinity
TRACE 15:44:35,765 marking as alive /127.0.0.1
TRACE 15:44:35,765 Updating heartbeat state version to 138 from 137 for /127.0.0.1 ...
TRACE 15:44:35,765 Scanning for state greater than 10 for /127.0.0.11
TRACE 15:44:35,766 Scanning for state greater than 10 for /127.0.0.11
TRACE 15:44:35,766 Sending a GossipDigestAck2Message to /127.0.0.1
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:45
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:35,766 Received a GossipDigestAckMessage from /127.0.0.1
TRACE 15:44:35,767 Scanning for state greater than 10 for /127.0.0.11
TRACE 15:44:35,767 Scanning for state greater than 10 for /127.0.0.11
TRACE 15:44:35,767 Sending a GossipDigestAck2Message to /127.0.0.1
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:46
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:35,850 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:35,954 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,066 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,169 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,263 Received a GossipDigestSynMessage from /127.0.0.13
TRACE 15:44:36,263 reporting /127.0.0.13
TRACE 15:44:36,264 reporting /127.0.0.13
TRACE 15:44:36,265 Scanning for state greater than 10 for /127.0.0.11
TRACE 15:44:36,265 Scanning for state greater than 137 for /127.0.0.1
TRACE 15:44:36,266 @@@@ Size of GossipDigestAckMessage is 69
TRACE 15:44:36,267 Sending a GossipDigestAckMessage to /127.0.0.13
- Experiment not running, continue ===>GDSynVHandler
Sending message ::
-->ID:47
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK

TRACE 15:44:36,274 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,275 Received a GossipDigestAck2Message from /127.0.0.13
TRACE 15:44:36,275 reporting /127.0.0.13
TRACE 15:44:36,276 marking as alive /127.0.0.13
TRACE 15:44:36,276 Updating heartbeat state version to 41 from 39 for /127.0.0.13 ...
TRACE 15:44:36,378 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,481 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,585 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,688 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:36,747 Gossip Digests are : /127.0.0.11:1308955447:12 /127.0.0.1:1308955160:138 /127.0.0.12:1308955413:18 /127.0.0.11:1308955447:12 /127.0.0.13:1308955372:41 
TRACE 15:44:36,748 Sending a GossipDigestSynMessage to /127.0.0.13 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:48
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:36,749 Sending a GossipDigestSynMessage to /127.0.0.12 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:48
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:36,749 Performing status check ...
TRACE 15:44:36,750 PHI for /127.0.0.13 : 0.13575565249455715
TRACE 15:44:36,750 PHI for /127.0.0.1 : 0.2749651502717491
TRACE 15:44:36,750 PHI for /127.0.0.12 : Infinity
TRACE 15:44:36,754 Received a GossipDigestAckMessage from /127.0.0.13
TRACE 15:44:36,755 Scanning for state greater than 11 for /127.0.0.11
TRACE 15:44:36,755 Scanning for state greater than 11 for /127.0.0.11
TRACE 15:44:36,756 Sending a GossipDigestAck2Message to /127.0.0.13
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:49
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:36,790 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,894 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:36,998 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:37,102 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:37,205 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:37,309 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:37,412 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:37,517 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:37,620 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:37,724 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
node1 recorded Tokensize 4
- Nodes not connected...
TRACE 15:44:37,747 Gossip Digests are : /127.0.0.11:1308955447:13 /127.0.0.12:1308955413:18 /127.0.0.1:1308955160:138 /127.0.0.11:1308955447:13 /127.0.0.13:1308955372:41 
TRACE 15:44:37,748 Sending a GossipDigestSynMessage to /127.0.0.1 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:50
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:37,750 Sending a GossipDigestSynMessage to /127.0.0.12 ...
- Experiment not running, continue ===>sendGossip
Sending message ::
-->ID:50
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_SYN

TRACE 15:44:37,751 Performing status check ...
TRACE 15:44:37,752 PHI for /127.0.0.13 : 0.4227327913121653
TRACE 15:44:37,752 PHI for /127.0.0.1 : 0.5518647287363647
TRACE 15:44:37,752 Received a GossipDigestAckMessage from /127.0.0.1
TRACE 15:44:37,754 reporting /127.0.0.1
TRACE 15:44:37,754 PHI for /127.0.0.12 : Infinity
TRACE 15:44:37,755 marking as alive /127.0.0.1
TRACE 15:44:37,755 Updating heartbeat state version to 140 from 138 for /127.0.0.1 ...
TRACE 15:44:37,756 Scanning for state greater than 11 for /127.0.0.11
TRACE 15:44:37,756 Scanning for state greater than 11 for /127.0.0.11
TRACE 15:44:37,756 Scanning for state greater than 39 for /127.0.0.13
TRACE 15:44:37,757 Sending a GossipDigestAck2Message to /127.0.0.1
- Experiment not running, continue ===>GDAckVHandler
Sending message ::
-->ID:51
FROM:/127.0.0.11
TYPE:GS
VERB:GOSSIP_DIGEST_ACK2

TRACE 15:44:37,833 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:37,936 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:38,040 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
TRACE 15:44:38,144 unable to connect to /127.0.0.12
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at java.net.Socket.<init>(Socket.java:375)
	at java.net.Socket.<init>(Socket.java:276)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody4(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.init$_aroundBody5$advice(OutboundTcpConnection.java:137)
	at org.apache.cassandra.net.OutboundTcpConnection.connect(OutboundTcpConnection.java:156)
	at org.apache.cassandra.net.OutboundTcpConnection.run(OutboundTcpConnection.java:89)
